pylmod/tests/common.py

```diff
@@ -0,0 +1,21 @@
+"""
+Base class and common constants needed for pylmod tests
+"""
+import os
+from unittest import TestCase
+
+
+class BaseTest(TestCase):
+    """
+    Base class with convenient constants and URL endpoints for pylmod testing.
```

Comment:
Uninformed question: This is the setup for unit testing.  How are the system integration tests performed?  I mean, they can't use these URLBASE and GBUUID constants.